plugins/grpc/node/v1.11.3/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# syntax=docker/dockerfile:1.4
+FROM node:18.12.1-bullseye AS build
+
+ARG TARGETARCH
+
+WORKDIR /build
+COPY --link package* .
+RUN /bin/bash <<'EOF'
+set -e
+# https://github.com/grpc/grpc-node/issues/1405#issuecomment-1195903603 - linux/arm64 binaries not available
```

Comment:
Still unresolved - will require Github actions runners that support linux/arm64.